### PR TITLE
Fixup mixin dashboards.

### PR DESCRIPTION
**Pull request by SuperQ**
* Add unit to panels.
* Add description to panels.

Signed-off-by: SuperQ <superq@gmail.com>

### CODE REVIEW

**Review by RichiH — APPROVED** [review]

File: snmp-mixin/dashboards/snmp_ubiquiti_wifi.json [hunk 1]
```diff
@@ -1259,6 +1271,7 @@
       "timeRegions": [],
       "timeShift": null,
       "title": "Traffic on air - $instance",
+      "description": "Taffic on air",
```
Comment (Nexucis) [inline snmp-mixin/dashboards/snmp_ubiquiti_wifi.json:1274]:
```suggestion
      "description": "Traffic on air",
```

File: snmp-mixin/dashboards/snmp_ubiquiti_wifi.json [hunk 2]
```diff
@@ -1593,6 +1609,7 @@
...
```
Comment (Nexucis) [inline snmp-mixin/dashboards/snmp_ubiquiti_wifi.json:1612]:
```suggestion
      "description": "Traffic on air",
```

File: snmp-mixin/dashboards/snmp_ubiquiti_wifi.json [hunk 3]
```diff
@@ -1927,6 +1947,7 @@
...
```
Comment (Nexucis) [inline snmp-mixin/dashboards/snmp_ubiquiti_wifi.json:1950]:
```suggestion
      "description": "Traffic on air",
```

File: snmp-mixin/dashboards/snmp_ubiquiti_wifi.json [hunk 4]
```diff
@@ -2261,6 +2285,7 @@
...
```
Comment (Nexucis) [inline snmp-mixin/dashboards/snmp_ubiquiti_wifi.json:2288]:
```suggestion
      "description": "Traffic on air",
```

File: snmp-mixin/dashboards/snmp_ubiquiti_wifi.json [hunk 5]
```diff
@@ -2595,6 +2623,7 @@
...
```
Comment (Nexucis) [inline snmp-mixin/dashboards/snmp_ubiquiti_wifi.json:2626]:
```suggestion
      "description": "Traffic on air",
```

File: snmp-mixin/dashboards/snmp_ubiquiti_wifi.json [hunk 6]
```diff
@@ -2929,6 +2961,7 @@
...
```
Comment (Nexucis) [inline snmp-mixin/dashboards/snmp_ubiquiti_wifi.json:2964]:
```suggestion
      "description": "Traffic on air",
```

**Review by Nexucis — COMMENTED** [review]
a bit late in the review, but just in case you want to know, here a list of things to fix

**Comment by SuperQ** [comment]
@Nexucis yea, the actual dashboard needs a lot of work. I was just trying to satisfy the linter.

I'm tempted to delete it, as it's a raw JSON dump, and unmaintained.

**Comment by RichiH** [comment]
I don't use SNMP a lot any more. I would prefer to help people who still
use it down their setups.

Sent by mobile; please excuse my brevity.

**Comment by Nexucis** [comment]
if it's unmaintained, perhaps we should delete and announce it somehow how bad it is specially if @RichiH you are saying you are encouraging people moving out of snmp exporter.

**Pull request merged**